controllers/BooksShowController.js

```diff
@@ -0,0 +1,52 @@
+angular
+  .module('booksApp')
+  .controller('BooksShowController', BooksShowController);
+
+var db = "https://super-crud.herokuapp.com/books"
```

Comment:
Also, this looks like a constant to me, so I would suggest using ALL_CAPS_AND_UNDERSCORES to name it. (i.e. DB or DB_URL rather than db)